applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -726,19 +964,33 @@ namespace Kratos
         const bool CalculateResidualVectorFlag
         )
     {
-       KRATOS_ERROR << "You are calle to the CalculateAll from the base class for solid elements" << std::endl; 
+       KRATOS_ERROR << "You have called to the CalculateAll from the base class for solid elements" << std::endl; 
     }
     
     //***********************************************************************
     //***********************************************************************
 
     double BaseSolidElement::GetIntegrationWeight(
         const GeometryType::IntegrationPointsArrayType& ThisIntegrationMethod,
-        const unsigned int PointNumber,
+        const unsigned int point_number,
         const double detJ
         )
     {
-        return ThisIntegrationMethod[PointNumber].Weight() * detJ;
+        return ThisIntegrationMethod[point_number].Weight() * detJ;
+    }
+    
+    //************************************************************************************
+    //************************************************************************************
+    
+    void BaseSolidElement::UpdateKinematics(
```

Comment:
@loumalouomega, what do you need by 

"We can change the name, but this is important to reduce the duplication of code and making more general the implementation. Right now the Only stress you can get is the tensor and just the linear, calculated manually (instead of using the CL)" ??